prov/rxm/src/rxm_init.c

```diff
@@ -152,6 +154,17 @@ static int rxm_init_info(void)
 		rxm_info.tx_attr->inject_size = RXM_BUF_SIZE;
 	}
 	rxm_info.tx_attr->inject_size -= sizeof(struct rxm_pkt);
+
+	if (!fi_param_get_int(&rxm_prov, "buffer_cnt", &param)) {
+		if (!(param & (param - 1))) {
+			rxm_ep_buffer_cnt = param;
+		} else {
```

Comment:
You can use roundup_power_of_two() to automatically convert to a power a two and remove that restriction from the user

Reply:
I think roundup might exceed recv queue sizes in certain cases. Btw, do we need this to be a power of two?